tests/delete_regress/tests.py

```diff
@@ -299,7 +299,10 @@ def setUpTestData(cls):
         cls.l2 = Login.objects.create(description="l2", orgunit=cls.o2)
 
     @skipUnlessDBFeature("update_can_self_select")
-    def test_ticket_19102_annotate(self):
+    def test_delete_subquery_for_annotated(self):
+        """
+        Deletion using subqueries works for annotated queries (#19102).
+        """
```

Comment:
```suggestion
    def test_annotate(self):
```
As the ticket number is in the class description and due to the name and docstring of the test class, I think I would remove all the doc strings added here and have shorter test names `test_extra`, `test_select_related`, `test_only`

Reply:
I usually try to write test names as a full sentence,
I'll try to follow this instead.